hw/ip/sram_ctrl/doc/_index.md

```diff
@@ -9,12 +9,15 @@ This document specifies the functionality of the SRAM memory controller.
 The SRAM controller is a module that is a peripheral on the chip interconnect bus, and thus follows the [Comportability Specification]({{< relref "doc/rm/comportability_specification" >}}).
 
 
-The SRAM controller interfaces with the SRAM data and address scrambling device and provides CSRs for requesting the scrambling keys and reporting any parity errors that have occurred in the macro.
+The SRAM controller contains the SRAM data and address scrambling device and provides CSRs for requesting the scrambling keys and triggering the hardware initialization feature.
```

Comment:
i left the same note in the PR, can the initialization doubled as a "wipe" also?

Reply:
yes, it technically can. should say that explicitly here in the docs?

Reply:
I feel like it's worth just a one-sentence statement that it doesn't HAVE to be called at init time.

Reply:
ok good. with the current LFSR implementation it is probably good to not claim it as a "secure wipe" feature, because for that the PRNG quality would have to be improved (the state is SW visible here, so fwd/bkwd security would have to be taken into account).

Reply:
I have added a sentence mentioning that this can be used as a cheap wiping mechanism, with a note that this does not give you strong security guarantees.